maintenance/importConfig.ts

```diff
@@ -589,63 +595,64 @@ async function parseOzwProduct(
 
 				ret.paramInformation[id] = parsedParam;
 			}
-		} else {
-			const parsedParam = ret.paramInformation[param.index] ?? {};
-
-			// don't use ?? here, some fields could be empty strings and ?? operator
-			// will not work
-			parsedParam.label =
-				ensureArray(param.label)[0] || parsedParam.label;
-			parsedParam.description =
-				ensureArray(param.Help)[0] || parsedParam.description;
-			parsedParam.valueSize = param.size || parsedParam.valueSize || 1;
-			parsedParam.minValue = param.min || parsedParam.min || 0;
-			parsedParam.maxValue = param.max || parsedParam.max || 100;
-			parsedParam.readOnly = Boolean(param.read_only);
-			parsedParam.writeOnly = Boolean(param.write_only);
-			parsedParam.allowManualEntry = param.type !== "list";
-			parsedParam.defaultValue = param.value || parsedParam.value || 0;
-
-			if (param.units) {
-				parsedParam.unit = param.units;
-			}
+		}
 
-			// could have multiple translations, if so it's an array, the first
-			// is the english one
-			if (isArray(parsedParam.description)) {
-				parsedParam.description = parsedParam.description[0];
-			}
+		const parsedParam = ret.paramInformation[param.index] ?? {};
+
+		// don't use ?? here, some fields could be empty strings and ?? operator
+		// will not work
+		parsedParam.label = ensureArray(param.label)[0] || parsedParam.label;
+		parsedParam.description =
+			ensureArray(param.Help)[0] || parsedParam.description;
+		parsedParam.valueSize = param.size || parsedParam.valueSize || 1;
+		parsedParam.minValue = param.min || parsedParam.min || 0;
+		parsedParam.maxValue = isBitSet
+			? param.bitmask
+			: param.max || parsedParam.max || 100;
+		parsedParam.readOnly = Boolean(param.read_only);
+		parsedParam.writeOnly = Boolean(param.write_only);
+		parsedParam.allowManualEntry = param.type !== "list";
+		parsedParam.defaultValue = param.value || parsedParam.value || 0;
+
+		if (param.units) {
+			parsedParam.unit = param.units;
+		}
 
-			if (typeof parsedParam.description !== "string") {
-				parsedParam.description = "";
-			}
+		// could have multiple translations, if so it's an array, the first
+		// is the english one
+		if (isArray(parsedParam.description)) {
+			parsedParam.description = parsedParam.description[0];
+		}
 
-			if (typeof parsedParam.defaultValue !== "number") {
-				parsedParam.defaultValue = 0;
-			}
+		if (typeof parsedParam.description !== "string") {
+			parsedParam.description = "";
+		}
 
-			// some values have typos, this fixes them
-			if (typeof parsedParam.maxValue === "string") {
-				parsedParam.maxValue = parseInt(
-					parsedParam.maxValue.replace(/[^0-9]/g, ""),
-				);
-			}
+		if (typeof parsedParam.defaultValue !== "number") {
+			parsedParam.defaultValue = 0;
+		}
 
-			const items = ensureArray(param.Item);
+		// some values have typos, this fixes them
+		if (typeof parsedParam.maxValue === "string") {
+			parsedParam.maxValue = parseInt(
+				parsedParam.maxValue.replace(/[^0-9]/g, ""),
```

Comment:
1. This changes negative values to their absolute value
2. Don't we need to do this for minValue too?

Reply:
Will fix this